daemon/src/lib.rs

```diff
@@ -275,6 +279,11 @@ where
 
         tasks.add(endpoint_context.run(endpoint));
 
+        let (supervisor, _ping_address) = supervisor::Actor::new({
+            move || ping::Actor::new(endpoint_addr.clone(), PING_INTERVAL)
+        });
+        let _ping_supervisor = supervisor.create(None).spawn(&mut tasks);
```

Comment:
❓ Why do we not add this one to `Self`? 

🛠 I think we should keep this one around as well.